GVFS/GVFS.Platform.Windows/GVFS.Platform.Windows.csproj

```diff
@@ -45,8 +45,8 @@
       <HintPath>..\..\..\packages\Microsoft.Database.Isam.1.9.4\lib\net40\Esent.Isam.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProjectedFSLib.Managed, Version=1.0.19079.1, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
-      <HintPath>..\..\..\packages\$(ProjFSManagedPackage)\lib\net461\ProjectedFSLib.Managed.dll</HintPath>
+    <Reference Include="ProjectedFSLib.Managed, Version=1.0.19108.2, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=AMD64">
+      <HintPath>..\..\..\packages\Microsoft.Windows.ProjFS.1.0.19108.2\lib\net461\ProjectedFSLib.Managed.dll</HintPath>

```

Comment:
Not sure if this is a big deal, but I see that `$(ProjFSManagedPackage)` was replaced with a hardcoded `Microsoft.Windows.ProjFS.1.0.19108.2`.  If this was done automatically by the IDE when we upgraded I think that's okay.

(Noticed this in a few files)

Reply:
It was the IDE.  Since we'll likely use the IDE to upgrade next time I think it's OK change, like you are suggesting.